data/core.yaml

```diff
@@ -552,284 +552,158 @@ en:
-    relations: |
-      # Relations
-
-      A relation is a special type of feature in OpenStreetMap that groups together
-      other features. For example, two common types of relations are *route relations*,
-      which group together sections of road that belong to a specific freeway or
-      highway, and *multipolygons*, which group together several lines that define
-      a complex area (one with several pieces or holes in it like a donut).
-
-      The group of features in a relation are called *members*. In the bottom of the
-      sidebar, you can see which relations a feature is a member of, and click on a
-      relation there will select it. When the relation is selected, you can see all of
-      its members listed in the sidebar and highlighted on the map.
-
-      For the most part, iD will take care of maintaining relations automatically
-      while you edit. The main thing you should be aware of is that if you delete a
-      section of road to redraw it more accurately, you should make sure that the
-      new section is a member of the same relations as the original.
-
-      ## Editing Relations
-
-      If you want to edit relations, here are the basics.
-
-      To add a feature to a relation, select the feature, click the "+" button in the
-      "All relations" section of the sidebar, and select or type the name of the relation.
-
-      To create a new relation, select the first feature that should be a member,
-      click the "+" button in the "All relations" section, and select "New relation...".
-
-      To remove a feature from a relation, select the feature and click the trash
-      button next to the relation you want to remove it from.
-
-      You can create multipolygons with holes using the "Merge" tool. Draw two areas (inner
-      and outer), hold the Shift key and click on each of them to select them both, and then
-      press the 'C' shortcut key. Other option is to select both, and then right-click one
-      of the and click the "Merge" (+) button.
+    help:
+      title: Help
+      welcome: "Welcome to the iD editor for [OpenStreetMap](https://www.openstreetmap.org/). With this editor you can update OpenStreetMap right from your web browser."
+      open_data_h: "### Open Data"
+      open_data: "Edits that you make on this map will be visible to everyone who uses OpenStreetMap. Your edits can be based on personal knowledge, on-the-ground surveying, or imagery collected from aerial or street level photos. Copying from commercial sources, like Google Maps, [is strictly forbidden](https://www.openstreetmap.org/copyright)."
+      before_start_h: "### Before you start"
+      before_start: "You should be familiar with OpenStreetMap and this editor before you start editing. iD contains a builtin walkthrough to teach you the basics of editing OpenStreetMap. Click 'Start the Walkthrough' on this screen to take the tutorial - it takes only about 15 minutes."
```

Comment:
Since this is for the base American English localization, we should probably use double quotation marks consistently. The British English localization can use single quotation marks as usual.